src/SFML/Network/Ftp.cpp

```diff
@@ -463,7 +463,7 @@ Ftp::Response Ftp::getResponse()
                         }
 
                         // Save the remaining data for the next time getResponse() is called
-                        m_receiveBuffer.assign(buffer + in.tellg(), length - in.tellg());
+                        m_receiveBuffer.assign(buffer + static_cast<std::streamoff>(in.tellg()), length - static_cast<std::size_t>(in.tellg()));
```

Comment:
The first cast should not be needed.

Reply:
Visual Studio 2015 and 2017 32bits [still complain about it](https://ci.sfml-dev.org/builders/windows-vc14-32/builds/506/steps/compile/logs/warnings%20%2816%29).

According to [cppreference](http://en.cppreference.com/w/cpp/io/streamoff) `std::streamoff` is usually typedef-ed to `long long` and for the pointer manipulation you need an `int` thus there needs to be an explicit cast.

Reply:
Ah, sorry I though it was the same type (std::streamoff vs std::streampos). And why not std::size_t by the way?

Reply:
I picked `std::streamoff` since that's what the compiler expected and converts to. I could try `std::size_t` as well, even though that will be `unsigned int` and not `int`, wouldn't it?

Reply:
The first argument of assign is a const char*, which `buffer + <any integral type>` should produce no matter what the integral type is. So I don't get why we need a std::streamoff specifically.

Reply:
Changed it to `std::size_t`.  🙂

Reply:
:+1: